testdata/serviceconfig.yml

```diff
@@ -30,19 +30,10 @@ rules:
           response_code: response.code | 200
   - kind: access-logs
     params:
-      logName: "access_log"
+      logName: access_log
       log:
-        logFormat: 1 # COMMON
-        templateExpressions:
```

Comment:
It's there, the github diff doesn't render it cause it's unchanged from before. If you expand the file down you should see the labels.